impeller/golden_tests/metal_screenshotter.mm

```diff
@@ -37,8 +37,11 @@
     return {};
   }
 
-  CIImage* ciImage = [[CIImage alloc] initWithMTLTexture:metal_texture
-                                                 options:@{}];
+  CGColorSpaceRef color_space = CGColorSpaceCreateDeviceRGB();
```

Comment:
Rather than create a device dependent color space which might vary over time as we change our CI hardware, should we perhaps create a standardized color space such as with one of these constants?

https://developer.apple.com/documentation/coregraphics/cgcolorspace/1408871-srgb

Reply:
Or Linear sRGB: https://developer.apple.com/documentation/coregraphics/cgcolorspace/1644734-linearsrgb

Reply:
I'm not sure if that would be appropriate.  It may be more appropriate that the colorspace matches the device one since that's what the texture will reflect?  I'd rather keep this similar to vulkan and as written.  If it ever breaks, we'll know quickly.

Reply:
If the machines in the CI lab are homogenous then it might not cause problems now, but if we ever go through a transition with different types of machines that happen to disagree on their native colorspace we might have some headaches at that time.